### DISCUSSION TITLE
Desktop App Releases

### DISCUSSION THREAD

**Discussion by trymeouteh** [hoppscotch/hoppscotch#3992] [Ideas]
I would like to see the desktop app be released in the following ways...

- Windows
   - EXE/MSI (DONE)
   - Scoop
   - Winget
- MacOS
   - DMG (DONE)
   - Homebrew
- Linux
   - AppImage (DONE)
   - NIX
   - Flatpak

**Comment by ShyVortex** [comment]
If you're interested, I've made a Flatpak packaging of Hoppscotch: https://github.com/ShyVortex/hoppscotch-flatpak

**Comment by Anifyuli** [comment]
I love Hoppscotch release with RPM format
